deploy/helm-chart/kube-mail/values.yaml

```diff
@@ -16,6 +16,7 @@ containerSecurityContext:
   enabled: false
   runAsUser: 1001
   runAsNonRoot: true
+policyClassName: {}
```

Comment:
This default value suggests that a user should supply an object as a value, while indeed a string is expected. Also, it should be `priorityClassName` and not `policyClassName`.

```suggestion
# priorityClassName: "system-cluster-critical"
```